Makefile

```diff
@@ -185,6 +185,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
+run-ci:
+	$(PYTHON) ./configure $(CONFIG_FLAGS)
```

Comment:
PYTHON is defined differently on different platforms. On some Jenkins slaves, this needs to be set explicitly via the environment variable.